API/parseClassToStructs.m

```diff
@@ -131,7 +122,7 @@
 problemDefCells{11} = inputStruct.nbairNames;
 problemDefCells{12} = inputStruct.nbsubNames;
 problemDefCells{13} = inputStruct.resolParNames;
-problemDefCells{14} = customFiles';
+problemDefCells{14} = customFiles;
```

Comment:
Might as well change this now to "inputStruct.files" and remove the "customFiles = inputStruct.files;" line above.